query/src/main/java/org/infinispan/query/SearchManager.java

```diff
@@ -72,6 +56,8 @@
     */
    void purge(Class<?> entityType);
 
+   void refresh(Class<?> entityType);
```

Comment:
What does refresh do?

Reply:
It forces re-creating index readers. That's only useful if near-real-time is enabled.

See https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#backend-lucene-io-refresh